### ISSUE TITLE
MaskVertexSet + MaskEdgeSet missing isEmpty implementation

### ISSUE THREAD

**Issue by omerb4** [jgrapht/jgrapht#876]
```
 * JGraphT version: 1.2.0
 * Java version (java -version)/platform:  11.0.4
```

**Issue**
isEmpty implementation is calculating the size of the set which takes too long while all it needs is to ask the iterator if he doesn't hasNext.

![image](https://user-images.githubusercontent.com/15855765/75230638-b1ff7b00-57bc-11ea-9577-95782a7c5566.png)

**Steps to reproduce (small coding example)**

**Expected behaviour**

**Other information**

**Comment by d-michail** [comment]
Contributions are welcomed

**Comment by omerb4** [comment]
Opened a PR:
https://github.com/jgrapht/jgrapht/pull/877

**Comment by Toptachamann** [comment]
Resolved in #877.
